Mail/Views/Search/SearchView.swift

```diff
@@ -90,6 +90,9 @@ struct SearchView: View {
                 CloseButton {
                     Constants.globallyResignFirstResponder()
                     mainViewState.isShowingSearch = false
+                    Task {
+                        await viewModel.mailboxManager.clearSearchResults()
```

Comment:
Should we do it on the `.onDisappear` function?

Reply:
We can't because the SearchView is not presented modally